<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEditWorld</class>
 <widget class="QDialog" name="DialogEditWorld">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>World Edit</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditName"/>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Depth Sorted Bin:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinBoxDepthSortedBin</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QSpinBox" name="spinBoxDepthSortedBin">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lineEditFile"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineEditName</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Latitude [deg]:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinBoxLat</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="spinBoxLat">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-90.000000000000000</double>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="pushButtonBrowse">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="spinBoxLon">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QCheckBox" name="checkBoxDepthSortedBin">
     <property name="text">
      <string>Depth Sorted Bin</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Longitude [deg]:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinBoxLon</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>File:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineEditFile</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>379</width>
       <height>205</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="spinBoxAlt">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Altitude [m]:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinBoxAlt</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>checkBoxDepthSortedBin</tabstop>
  <tabstop>spinBoxDepthSortedBin</tabstop>
  <tabstop>spinBoxLat</tabstop>
  <tabstop>spinBoxLon</tabstop>
  <tabstop>spinBoxAlt</tabstop>
  <tabstop>lineEditFile</tabstop>
  <tabstop>pushButtonBrowse</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogEditWorld</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogEditWorld</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
